transforms/github/changes.yaml

```diff
@@ -0,0 +1,8 @@
+# pretend the following PRs were merged
+include-prs:
+  - https://github.com/github/advisory-database/pull/2630
+
+# assuming the following GHSAs were reviewed, even if they
+# are not returned by the API
+assume-reviewed:
```

Comment:
From Josh: This mechanism probably isn't needed. If we're going to include an unreviewed GHSA, we're going to have to change it anyway. So just have a mechanism for changing fetched GHSA data, and assume any changed GHSA should be included in our output, regardless of whether GHSA has reviewed it.